### DISCUSSION TITLE
Layout 的多标签模式，当标签超过屏幕宽度不会自动右移

### DISCUSSION THREAD

**Discussion by gudufy** [dotnetcore/BootstrapBlazor#3758] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

![微信截图_20240629070236](https://github.com/dotnetcore/BootstrapBlazor/assets/117419/7a663f10-7cd8-4c14-ba3d-90d84e1fb62d)
还有个问题就是同样的url，参数不同则会生成多个tab，比如/users和/users?a=3会生成两个tab，能否控制不同参数都在一个tab里显示？

### Expected Behavior

1.当标签超过屏幕宽度自动右移，且增加鼠标滚轮控制左右移动
2.同一个url不同参数都在一个tab里显示

### Steps To Reproduce

在简约项目里打开多标签模式，然后增加以下tab
`var menus = new List<MenuItem>
{
    new() { Text = "Index", Icon = "fa-solid fa-fw fa-flag", Url = "/" , Match = NavLinkMatch.All},
    new() { Text = "Counter", Icon = "fa-solid fa-fw fa-check-square", Url = "/counter" },
    new() { Text = "Weather", Icon = "fa-solid fa-fw fa-database", Url = "/weather" },
    new() { Text = "Table", Icon = "fa-solid fa-fw fa-table", Url = "/table" },
    new() { Text = "花名册", Icon = "fa-solid fa-fw fa-users", Url = "/users" },
    new() { Text = "Index", Icon = "fa-solid fa-fw fa-flag", Url = "/?a=1" , Match = NavLinkMatch.All},
    new() { Text = "Counter", Icon = "fa-solid fa-fw fa-check-square", Url = "/counter?a=1" },
    new() { Text = "Weather", Icon = "fa-solid fa-fw fa-database", Url = "/weather?a=1" },
    new() { Text = "Table", Icon = "fa-solid fa-fw fa-table", Url = "/table?a=1" },
    new() { Text = "花名册", Icon = "fa-solid fa-fw fa-users", Url = "/users?a=1" },
    new() { Text = "Index", Icon = "fa-solid fa-fw fa-flag", Url = "/?a=2" , Match = NavLinkMatch.All},
    new() { Text = "Counter", Icon = "fa-solid fa-fw fa-check-square", Url = "/counter?a=2" },
    new() { Text = "Weather", Icon = "fa-solid fa-fw fa-database", Url = "/weather?a=2" },
    new() { Text = "Table", Icon = "fa-solid fa-fw fa-table", Url = "/table?a=2" },
    new() { Text = "花名册", Icon = "fa-solid fa-fw fa-users", Url = "/users?a=2" },
    new() { Text = "Index", Icon = "fa-solid fa-fw fa-flag", Url = "/?a=3" , Match = NavLinkMatch.All},
    new() { Text = "Counter", Icon = "fa-solid fa-fw fa-check-square", Url = "/counter?a=3" },
    new() { Text = "Weather", Icon = "fa-solid fa-fw fa-database", Url = "/weather?a=3" },
    new() { Text = "Table", Icon = "fa-solid fa-fw fa-table", Url = "/table?a=3" },
    new() { Text = "花名册", Icon = "fa-solid fa-fw fa-users", Url = "/users?a=3" }
};`

### .NET Version

.net 8

### Anything else?

vs 2022

**Comment by bb-auto** [comment]
@gudufy Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment]
@gudufy 感谢提交 Issue 右上角帮忙点一下 star 谢谢啦，请尽快补充复现工程

**Reply by gudufy** [reply]
> @gudufy 感谢提交 Issue 右上角帮忙点一下 star 谢谢啦，请尽快补充复现工程

就在https://www.blazor.zone/layout-page这个页在多加一些菜单然后点开就可以复现了。

**Reply by ArgoZhang** [reply]
@gudufy 已修复请查看 #3760 相关 Issue
